_episodes/03-loop.md

```diff
@@ -192,33 +192,34 @@ of the loop body (e.g. `end for`); what is indented after the `for` statement be
 Here's another loop that repeatedly updates a variable:
 
 ~~~
+names = ['Curie', 'Darwin', 'Turing']
 length = 0
-for vowel in 'aeiou':
+for value in names:
     length = length + 1
-print('There are', length, 'vowels')
+print('There are', length, 'names in the list.')
 ~~~
 {: .language-python}
 
 ~~~
-There are 5 vowels
+There are 3 elements in the list.
```

Comment:
`elements` or `names` or ...?

Reply:
Yep, thanks for catching that, it's supposed to be names :)